vault-core/src/main/java/org/apache/jackrabbit/vault/fs/io/AbstractExporter.java

```diff
@@ -201,6 +201,8 @@ public void export(VaultFile parent, boolean noClose)
             // update properties
             setProperty(MetaInf.CREATED, Calendar.getInstance());
             setProperty(MetaInf.CREATED_BY, mgr.getUserId());
+            String moduleName = "org.apache.jackrabbit.vault";
+            setProperty(MetaInf.GENERATOR, moduleName + ":" + getVersion(moduleName, this.getClass()));
```

Comment:
Please also add a dedicated getter in `org.apache.jackrabbit.vault.packaging.PackageProperties`

Reply:
For the `generator` string?

Reply:
The high-level API (with individual getters) is the preferred way of accessing package properties rather than using the low-level API (accessing individual keys) as 
a) it doesn't require to deal with key literals or constants
b) allows for caching or lookup in the manifest alternatively